webrtc.html

```diff
@@ -6278,6 +6279,17 @@ <h2>
                   This carries the <code class="ice">ufrag</code> as defined in
                   section 15.4 of [[RFC5245]].
                 </dd>
+                <dt>
+                  <dfn>relayProtocol</dfn> of type <span class=
+                  "idlMemberType">DOMString</span>
+                </dt>
+                <dd>
+                  <p>
+                    It is the protocol used by the endpoint to communicate with the TURN server. This
+                    is only present for local candidates. Valid values are <code>"udp"</code>,
```

Comment:
Should this say "is only present for local TURN candidates"?

Reply:
I copied from webrtc-stats but this seems like a sensible improvement, will update both